engines/director/lingo/lingo-builtins.cpp

```diff
@@ -2521,8 +2521,8 @@ void LB::b_immediateSprite(int nargs) {
 			if (sc->getNextFrame() && !sp->_immediate) {
 				// same as puppetSprite
 				Channel *channel = sc->getChannelById(sprite.asInt());
-
-				channel->replaceSprite(sc->_frames[sc->getNextFrame()]->_sprites[sprite.asInt()]);
+				// TODO: Fix logic here for next sprite
```

Comment:
write `warning("STUB: Fix logic here for next sprite")`. It should be prominent.

And in general, this b_immedateSprite() should be reviewed. I am not sure it is implemented correctly.

If it matched with `the immediate of sprite`, then it could go down to setting the `immediate` flag. More investigation is required. This Lingo is used in The Apartment in two movies.